flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
I forgot to remove this after I made it build from source. Before it got the deb package with x86_64 architecture